client/index.css

```diff
@@ -199,3 +199,119 @@ ul {
 .chat ul li.typing div {
     animation: flash 2s infinite;
 }
+#settings {
+    position: absolute;
+    top: 40px;
+    left: 0;
+    right: 0;
+    width: 345px;
+    height: auto;
+    margin: auto;
+}
+#settings #icon-username {
+    position: absolute;
+    top: 30px;
+    left: 100px;
+    width: 100px;
+    height: auto;
+}
+#icon-username h2 {
+    position: absolute;
+    top: -20px;
+    left: 120px;
+}
+#settings .icon {
+    position: absolute;
+    width: 100px;
+    height: 150px;
+    border: 1px solid black;
+}
+.icon img {
+    position: absolute;
+    height: 100px;
+    width: 100px;
+}
+.icon span {
+    position: absolute;
+    top: 100px;
+    width: 100%;
+    padding: 5px 10px;
+    border-top: 1px solid black;
+    cursor: pointer;
+}
+#settings #form {
+    position: absolute;
+    top: 225px;
+    width: 400px;
+}
+#form select {
+    width: 174px;
+    display: inline-block;
+    padding: 0 0 0 9px;
+}
+#form select.day,
+#form select.month {
+    width: 50px;
+}
+#form select.year {
+    width: 64px;
+}
+#form select.month,
+#form select.year {
+    margin-left: 5px;
+}
+#form label {
+    width: 150px;
+    margin: 0px 20px 30px 0px;
+    text-align: right;
+}
+#form input {
+    width: 173px;
+    display: inline-block;
+}
+#form button {
+    position: relative;
+    left: 150px;
+    color: white;
+    background-color: #0084ff;
+    border: 1px solid #0084ff;
+    border-radius: 5px;
+    padding: 10px 20px;
+}
+.top img.avatar {
+    position: absolute;
+    top: 10%;
+    right: 22px;
+    border-radius: 50px;
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+}
+.top #menu {
+    position: absolute;
+    top: 50px;
+    right: 24px;
+    background-color: LightSlateGray;
+    border-radius: 5px;
+    padding: 0;
+    z-index: 2;
+}
+.top #menu:before {
+    content: '';
```

Comment:
It's possible that your changes broke utf-8 encoding. Are you sure you are saving your files as utf-8? See screenshot.

<img width="289" alt="screen shot 2017-07-23 at 05 53 17" src="https://user-images.githubusercontent.com/544572/28496100-52769dda-6f6b-11e7-93e8-7643f718d892.png">